docs/open-source-project/porting/system.md

```diff
@@ -100,6 +100,14 @@ To use most of the `resourced` functionalities, you must enable the following `c
 >
 > To use the `resourced` freezer feature, you must install the freezer plugin by enabling `CONFIG_FREEZER`.
 
+Resourced supports several swapping scenarios to provide more RAM to applications running in the foreground. To support them, enable the following options in the kernel configuration:
+
+  - `CONFIG_MEMCG_SWAP`: control swapping applications' memory, `CONFIG_SWAP` will be enabled automatically
+  - `CONFIG_ZRAM`: compressed block device
+  - `CONFIG_ZRAM_LZ4_COMPRESS`: an alternative compression algorithm, by default ZRAM uses LZO
+  - `CONFIG_ZSWAP`: compress memory before or dumping it to a storage device
+  - `CRYPTO_DEFLATE`, `CRYPTO_ZLIB`, `CRYPTO_LZO`, `CRYPTO_LZ4`, `CRYPTO_LZ4HC`: compression algorithms available as a part of kernel cyrpto API
```

Comment:
Please rephrase to:

Compression algorithms available as part of the kernel crypto API.

Reply:
Done.